hack/logcheck.out

```diff
@@ -127,16 +127,16 @@
 /pkg/tunneler/listener.go:211:2: function "V" should not be used, convert to contextual logging
 /pkg/tunneler/listener.go:82:4: function "Infof" should not be used, convert to contextual logging
 /pkg/tunneler/listener.go:82:4: function "V" should not be used, convert to contextual logging
-/pkg/tunneler/tunnel.go:146:3: function "InfoS" should not be used, convert to contextual logging
-/pkg/tunneler/tunnel.go:146:3: function "V" should not be used, convert to contextual logging
-/pkg/tunneler/tunnel.go:178:5: function "Infof" should not be used, convert to contextual logging
-/pkg/tunneler/tunnel.go:178:5: function "V" should not be used, convert to contextual logging
-/pkg/tunneler/tunnel.go:182:4: function "Infof" should not be used, convert to contextual logging
-/pkg/tunneler/tunnel.go:182:4: function "V" should not be used, convert to contextual logging
-/pkg/tunneler/tunnel.go:195:4: function "Infof" should not be used, convert to contextual logging
-/pkg/tunneler/tunnel.go:195:4: function "V" should not be used, convert to contextual logging
-/pkg/tunneler/tunnel.go:230:4: function "Infof" should not be used, convert to contextual logging
-/pkg/tunneler/tunnel.go:230:4: function "V" should not be used, convert to contextual logging
+/pkg/tunneler/tunnel.go:147:3: function "InfoS" should not be used, convert to contextual logging
```

Comment:
/hold

At this point we're months in, and I think it's not appropriate to be deferring this any longer. You're touching the tunneler code, so please change these log lines to contextual logging in order to merge.

Reply:
Thanks, done!